testing/test_base

```diff
@@ -9,6 +9,7 @@ export TEST_RESULTS
 
 testing/test_check
 testing/test_topology
+testing/test_flat cleanup
```

Comment:
Best to keep the arg syntax same as the others -- so no "cleanup" arg here... Just default to cleaning. If there's a reason to not cleanup then I'd go with the inverse with "noclean" or similar.

Reply:
Ok, changed the variable to `no_clean`in testing/test_flat and removed `cleanup` here